components/channel_notifications_modal/components/expand_view.tsx

```diff
@@ -15,6 +14,18 @@ import SettingItemMax from 'components/setting_item_max.jsx';
 import Describe from './describe.jsx';
 import ExtraInfo from './extra_info.jsx';
 import SectionTitle from './section_title.jsx';
+type Props = {
```

Comment:
nit : extra line between

Reply:
Would love a rule for this in eslint.

Reply:
yup we will add it. thank for suggesting